go.mod

```diff
@@ -139,3 +140,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
+
+replace github.com/operator-framework/catalogd => github.com/openshift/operator-framework-catalogd v0.0.0-20241018021036-4275eb2ee38d
```

Comment:
Is this temporary? If not how do we place to keep this updated?

Reply:
My thinking was that this would need to be kept around, but I realize we also didn't do this replacement for the operator-controller dependency we have.

Is there a particular reason we feel comfortable using the upstream vs a specific commit in the corresponding downstream repo? I would expect us to want to ensure everything downstream is properly synced and referencing the corresponding commits for the same OCP version. If we use the upstream and make a downstream only change to something like our group (i.e going from `olm.operatorframework.io` to `olm.openshift.io`) cluster-olm-operator would break.

I do realize this is likely to get stale pretty quickly, but for keeping it up to date I would expect us to update our downstream sync tooling to also know how and when to update these replacements such that everything is in sync for our downstream uses. For backporting fixes, this would likely require manual updates.

Reply:
Consensus is cross this bridge when we get to it. Will remove this replacement.

Reply:
I think the 'why upstream versus downstream' is a larger conversation, but without the tools already updated to implement whatever we agree we need to do, this replace seems premature.

Reply:
Removed in https://github.com/openshift/cluster-olm-operator/pull/73/commits/5b237d4e4d462f19ff7571661b839393bde6b676